src/Og.php

```diff
@@ -100,17 +105,32 @@ public static function createField($plugin_id, $entity_type, $bundle, array $set
       ]);
     }
 
-    $widget = $form_display->getComponent($plugin_id);
-    $widget['type'] = 'og_complex';
-    $widget['settings'] = [
-      'match_operator' => 'CONTAINS',
-      'size' => 60,
-      'placeholder' => '',
-    ];
+    $form_display_definition = $og_field->getFormDisplayDefinition($settings['form_display']);
+
 
-    $form_display->setComponent($plugin_id, $widget);
+    $form_display->setComponent($plugin_id, $form_display_definition);
     $form_display->save();
 
+
+    // Set the view display for the "default" and "teaser" types.
+    $view_display_definition = $og_field->getViewDisplayDefinition($settings['view_display']);
```

Comment:
This comment is only applicable for nodes when using the "standard" installation profile. The "teaser" view display is not present for other entity types and when using other profiles.